cmake/Blas.cmake

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same here. Is this a copy of what's in mshadow?  If so, can it have a different file name and make sure mshadow doesn't load its own from its CMakeLists.txt or other module file? (how is that prevented right now?)